lms/djangoapps/discussion/tasks.py

```diff
@@ -206,10 +208,14 @@ def _build_message_context(context):  # lint-amnesty, pylint: disable=missing-fu
     message_context.update(context)
     thread_author = User.objects.get(id=context['thread_author_id'])
     comment_author = User.objects.get(id=context['comment_author_id'])
+    post_link = _get_mfe_thread_url(context) if ENABLE_DISCUSSIONS_MFE_FOR_EVERYONE.is_enabled(
```

Comment:
@saadyousafarbi We also need to check for the without iframe flag. The email should only send out the MFE link in case we are rendering it without the iframe.